proto/xudrpc.proto

```diff
@@ -467,6 +467,17 @@ message Order {
   uint64 hold = 10 [json_name = "hold"];
 }
 
+message Currency {
```

Comment:
I was going to say that I think it would be better to use the `SwapClient` enum we already have defined above in the proto definition for the `swap_client` field, but then I realized that the structure of this message is actually identical to the `AddCurrencyRequest` message. So I think it would be best to actually just rename `AddCurrencyRequest` to `Currency` and then use it for both the `AddCurrency` request and the `ListCurrencies` response. That way we don't have to maintain two message definitions for the same thing.

For the enum swap client field, in the cli you can print the string representation.